nevergrad/optimization/optimizerlib.py

```diff
@@ -1217,7 +1217,7 @@ def __init__(self, instrumentation: Union[int, Instrumentation], budget: Optiona
     def _optimizers(self) -> List[base.Optimizer]:
         if not self._optimizers_:
             self._optimizers_ = []
-            conv = {"num_workers": self.num_workers, "dimension": self.dimension}
+            conv = {"num_workers": self.num_workers, "dimension": self.dimension, "sqrt": int(np.sqrt(self.budget)) if self.budget else self.num_workers}
```

Comment:
I know it isn't part of this commit but conv really makes me think of convolution while I guess it means conversion here ? Maybe we should call it conversion or budget_conversion instead (or call a method called compute_budget(budget:str) instead).

Reply:
i chose "converter" and will let Jeremy make a decision for this.